pkg/cosign/tlog.go

```diff
@@ -390,13 +404,17 @@ func FindTlogEntry(ctx context.Context, rekorClient *client.Rekor,
 	return results, nil
 }
 
-// VerityTLogEntry verifies a TLog entry.
-// The argument *client.Rekor is unused and may be nil.
-func VerifyTLogEntry(ctx context.Context, _ *client.Rekor, e *models.LogEntryAnon) error {
+// VerityTLogEntry verifies a TLog entry against a map of trusted rekorPubKeys indexed
+// by log id.
+func VerifyTLogEntry(e *models.LogEntryAnon, rekorPubKeys *TrustedRekorPubKeys) error {
```

Comment:
Nit: maybe `func VerifyTLogEntryOffline`?

Reply:
Might as well since we're changing API. Online only means FindTlogEntry(Online) and run it through the same procedure as offline verification right now. We don't hold any client state to really do an "online verification"